.github/workflows/lint-test-v2.yaml

```diff
@@ -0,0 +1,72 @@
+name: Lint and Test Charts v2
```

Comment:
you may want to consider NOT adding a separate workflow, but introducing a matrix in the main workflow with `version: v1 | v2` parameter and only changing the main `ct` step based on that parameter.

Example: https://github.com/jaegertracing/jaeger/blob/acbc0e1f523a36c2080dd4f4544ef6ef66216e86/.github/workflows/ci-e2e-kafka.yml#L24

Reply:
yes i did that